src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" action="https://www.google.com/">
+      <div class="search-bar-big">
```

Comment:
Redundant div

Reply:
not fixed, remove this div. also for another form

Reply:
delete `search-bar-big/small` divs